docs/source/introduction/quickstart.ipynb

```diff
@@ -22,45 +22,56 @@
     "\n",
```

Comment:
<p>maybe more general: "for example in optimization or as covariance matrices."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Where do you want to put this? After " Such knowledge.."? Its not clear from the NB reviewer. </p>

Reply:
<p>...similar linear systems need to be solved, e.g. in optimization.</p><p> </p>

Reply:
done